### ISSUE TITLE
Slack and MS Teams notification doesn't import via content pack

### ISSUE THREAD

**Issue by alexey-ban** [Graylog2/graylog-plugin-integrations#1113]
## Steps To Reproduce

1. Create Notification with type **slack-notification-v1** (built-in plugin since v4.2.1) or **teams-notification-v2** (built-in plugin since v4.3.2)
1. Create content pack with notification bellow
1. Try install content pack

## Environment

- Graylog Version: 4.3.3-1
- Elasticsearch Version: 7.x
- MongoDB Version: -
- Browser Version: -

## Some logs:

Browser:
![Screenshot from 2022-07-20 14-20-14](https://user-images.githubusercontent.com/20123880/179980847-8fe5515e-a73f-4bf5-aacb-9f4ea8183e4d.png)

server log:

```
Caused by: java.lang.IllegalArgumentException: Could not resolve type id 'slack-notification-v1' as a subtype of [simple type, class org.graylog.events.contentpack.entities.EventNotificationConfigEntity]: known type ids = [email-notification-v1, http-notification-v1, legacy-alarm-callback-notification-v1, teams-notification-v1] (for POJO property 'config')
 at [Source: UNKNOWN; line: -1, column: -1] (through reference chain: org.graylog.events.contentpack.entities.AutoValue_NotificationEntity$Builder["config"])
	at com.fasterxml.jackson.databind.ObjectMapper._convert(ObjectMapper.java:3751) ~[graylog.jar:?]
	at com.fasterxml.jackson.databind.ObjectMapper.convertValue(ObjectMapper.java:3669) ~[graylog.jar:?]
	at org.graylog.events.contentpack.facade.NotificationFacade.decode(NotificationFacade.java:99) ~[graylog.jar:?]
	at org.graylog.events.contentpack.facade.NotificationFacade.createNativeEntity(NotificationFacade.java:90) ~[graylog.jar:?]
	at org.graylog2.contentpacks.ContentPackService.installContentPack(ContentPackService.java:150) ~[graylog.jar:?]
	... 29 more
Caused by: com.fasterxml.jackson.databind.exc.InvalidTypeIdException: Could not resolve type id 'slack-notification-v1' as a subtype of [simple type, class org.graylog.events.contentpack.entities.EventNotificationConfigEntity]: known type ids = [email-notification-v1, http-notification-v1, legacy-alarm-callback-notification-v1, teams-notification-v1] (for POJO property 'config')
 at [Source: UNKNOWN; line: -1, column: -1] (through reference chain: org.graylog.events.contentpack.entities.AutoValue_NotificationEntity$Builder["config"])
	at com.fasterxml.jackson.databind.exc.InvalidTypeIdException.from(InvalidTypeIdException.java:43) ~[graylog.jar:?]
	at com.fasterxml.jackson.databind.DeserializationContext.invalidTypeIdException(DeserializationContext.java:1635) ~[graylog.jar:?]
	at com.fasterxml.jackson.databind.DeserializationContext.handleUnknownTypeId(DeserializationContext.java:1187) ~[graylog.jar:?]
	at com.fasterxml.jackson.databind.jsontype.impl.TypeDeserializerBase._handleUnknownTypeId(TypeDeserializerBase.java:291) ~[graylog.jar:?]
	at com.fasterxml.jackson.databind.jsontype.impl.TypeDeserializerBase._findDeserializer(TypeDeserializerBase.java:162) ~[graylog.jar:?]
	at com.fasterxml.jackson.databind.jsontype.impl.AsPropertyTypeDeserializer._deserializeTypedForId(AsPropertyTypeDeserializer.java:113) ~[graylog.jar:?]
	at com.fasterxml.jackson.databind.jsontype.impl.AsPropertyTypeDeserializer.deserializeTypedFromObject(AsPropertyTypeDeserializer.java:97) ~[graylog.jar:?]
	at com.fasterxml.jackson.databind.deser.AbstractDeserializer.deserializeWithType(AbstractDeserializer.java:254) ~[graylog.jar:?]
	at com.fasterxml.jackson.databind.deser.impl.MethodProperty.deserializeSetAndReturn(MethodProperty.java:167) ~[graylog.jar:?]
	at com.fasterxml.jackson.databind.deser.BuilderBasedDeserializer.vanillaDeserialize(BuilderBasedDeserializer.java:269) ~[graylog.jar:?]
	at com.fasterxml.jackson.databind.deser.BuilderBasedDeserializer.deserialize(BuilderBasedDeserializer.java:193) ~[graylog.jar:?]
	at com.fasterxml.jackson.databind.ObjectMapper._convert(ObjectMapper.java:3746) ~[graylog.jar:?]
	at com.fasterxml.jackson.databind.ObjectMapper.convertValue(ObjectMapper.java:3669) ~[graylog.jar:?]
	at org.graylog.events.contentpack.facade.NotificationFacade.decode(NotificationFacade.java:99) ~[graylog.jar:?]
	at org.graylog.events.contentpack.facade.NotificationFacade.createNativeEntity(NotificationFacade.java:90) ~[graylog.jar:?]
	at org.graylog2.contentpacks.ContentPackService.installContentPack(ContentPackService.java:150) ~[graylog.jar:?]
	... 29 more
```

**Comment by danotorrey** [comment]
Thanks for the report @alexey-ban. A fix has been drafted in https://github.com/Graylog2/graylog-plugin-integrations/pull/1129 and https://github.com/Graylog2/graylog2-server/pull/13171. The intention is to include the fix in an upcoming bug fix release.

**Comment by danotorrey** [comment]
@alexey-ban The fix was included in the recent Graylog `4.3.5` release.
